_posts/2021-11-08-october-rubygems-updates.md

```diff
@@ -0,0 +1,42 @@
+---
+title: October 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in October.
+
+
+## RubyGems News
+
+In October, We released new versions of RubyGems [3.2.29](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3229--2021-10-08) and [3.2.30](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3230--2021-10-26) and Bundler [2.2.29](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2229-october-8-2021) and [2.2.30](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2230-october-26-2021).
+
+In addition to these releases, we also worked on the following:
```

Comment:
Hi @lauragift21! Well, the changes bellow are included in the above releases, so I'd say something like this instead:

```suggestion
The following is a non-exhaustive list of the improvements included in the above releases (see changelogs for more details):
```